src/EnergyPlus/StateManagement.cc

```diff
@@ -332,6 +341,7 @@ void EnergyPlus::clearAllStates(OutputFiles &outputFiles)
     OutputReportTabularAnnual::clear_state();
     OutsideEnergySources::clear_state();
     PackagedTerminalHeatPump::clear_state();
+    Photovoltaics::clear_state();
```

Comment:
5 new clear_states added and called from the state reset manager.  And there are still more I'm sure.